source/architecture.rst

```diff
@@ -1,11 +1,73 @@
-Architecture of S-ENDA
-"""""""""""""""""
+Architecture for Sprint 1
```

Comment:
"Sprint 1" is not understandable for externals. I suggest rather to mention the sprint goal. What about "Architecture for S-ENDA find and access"?